.ci/test_r_package.sh

```diff
@@ -50,10 +50,25 @@ if [[ $AZURE != "true" ]] && [[ $OS_NAME == "linux" ]]; then
             texlive-fonts-extra \
             qpdf \
             || exit -1
+
+    # https://github.com/r-lib/actions/issues/111
+    if [[ $R_BUILD_TYPE == "cran" ]]; then
+        sudo apt-get install \
+            --no-install-recommends \
+            -y \
+                autoconf=$(cat R-package/AUTOCONF_UBUNTU_VERSION) \
+                devscripts \
```

Comment:
oh! Do you think it is worth still maintaining R parts of that Dockerfile, since all R jobs have been moved to GitHub Actions? In my opinion it's better to not do that unless we some day decide to run R jobs in Azure DevOps again

Reply:
Agree with you! Then could you please remove existing R part from the Dockerfile?

Reply:
Sure! I just added https://github.com/guolinke/lightgbm-ci-docker/pull/11